.tx/config

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+# TODO: Transifex integration
+#
+# 1) Create message catalog:
+#    $ python setup.py extract_messages
+#    $ python setup.py init_catalog -l <lang>
+#    $ python setup.py compile_catalog
+# 2) Ensure project has been created on Transifex under the inveniosoftware
+#    organisation.
+# 3) Install the transifex-client
+#    $ pip install transifex-client
+# 4) Push source (.pot) and translations (.po) to Transifex
+#    $ tx push -s -t
+# 5) Pull translations for a single language from Transifex
+#    $ tx pull -l <lang>
+# 6) Pull translations for all languages from Transifex
+#    $ tx pull -a
+
+[main]
+host = https://www.transifex.com
+
+[invenio.invenio-audit-logs-messages]
+file_filter = invenio_audit_logs/translations/<lang>/LC_MESSAGES/messages.po
+source_file = invenio_audit_logs/translations/messages.pot
```

Comment:
Its either this file is not needed, or we should explicitly add the translation path.